cypress/e2e/user-onboarding/user-onboarding-new.ts

```diff
@@ -49,7 +49,11 @@ describe("User Onboarding (new user)", () => {
 
     it("page is shown", () => {
         cy.get('.mx_UserOnboardingPage').should('exist');
-        cy.percySnapshot("User onboarding page");
+        cy.get('.mx_UserOnboardingList').should('exist');
+        // Wait for animation to finish
+        cy.wait(5_000);
+        cy.get('.mx_UserOnboardingPage')
+            .percySnapshotElement("User onboarding page");
```

Comment:
We should avoid waits wherever possible, if someone changes the animation speed it'll break the percy tests possibly without being spotted in the PR due to not all PRs running Percy. It needs the label applied